src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  text-transform: uppercase;
+  white-space: nowrap;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  position: relative;
+}
+
+.nav__list {
+  display: flex;
+  z-index: 1;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: inline-block;
+  margin: 0 10px;
+  width: fit-content;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  text-decoration: none;
+  color: #000;
+  transition: color 0.3s;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.nav__link:target {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
color '#00acdc' is duplicated, so please move all duplicated colors and sizes to the variables

Reply:
Moved all duplicated colors to variables